http_check/conf.yaml.example

```diff
@@ -124,6 +124,14 @@ instances:
     #
     # allow_redirects: true
 
+
+    # The (optional) include_default_header parameter can include the default headers.
+    # Default headers can be found at https://github.com/DataDog/dd-agent/blob/master/util.py#L54-L63
```

Comment:
In my opinion, "can include" is a a little misleading. I'd simply change it to ".. instructs the check to include the default headers."

The link also needs to be updated to pin a tag, not the master branch as it can change anytime. For example https://github.com/DataDog/dd-agent/blob/5.10.0/util.py#L58-L67